src/api/app/models/kiwi/repository.rb

```diff
@@ -17,6 +17,7 @@ class Kiwi::Repository < ApplicationRecord
   #### Scopes (first the default_scope macro if is used)
 
   #### Validations macros
+  validates :alias, uniqueness: { scope: :image }
```

Comment:
[nitpick] Seems like it would not trigger a build failure, however, it might make sense to also check for the uniqueness of the path. Doesn't really make sense to have two different repositories with the same path and would produce also "overhead" as you would query the packages several times etc etc

Reply:
Ok, I will add it BTW 😄

Reply:
Thank you very much!